docs/sources/configure/integrations/references/slack/index.md

```diff
@@ -0,0 +1,382 @@
+
+Grafana IRM allows you to customize the names of incident-specific Slack channels by setting prefixes, making it easier to organize and search for incident channels.
+To customize Slack channel prefixes:
+
+1. Click **Incident** in the left-side menu.
+1. Go to **Settings**.
+1. Scroll down to the **Prefixes** section in the Settings page.
+1. Click **+ Add Prefix** and provide a name and description.
+1. Edit any existing prefixes as needed, then click **Update**.
+1. Once your prefixes are defined, you can choose which prefix to use when declaring an incident in Grafana Incident.
+
+### Configure Incident announcements
+
+Incident announcements help keep your team aware and informed during critical events.
+Configure these announcements to ensure that stakeholders are kept up-to-date in Slack during an incident.
+
+To configure Incident announcements:
+
+1. Navigate to **Incident** in the left-side menu, then click **Integrations**.
+1. Click **Slack IRM** to open the Slack IRM integration page.
+1. Select the Slack channel where you want to send notifications.
+You can either choose from the dropdown menu or manually add your Channel ID. The Channel ID can be found in the Slack channel’s **About** tab.
+1. Define the incident-specific fields:
+
+- **Incident type**: Choose whether the incident is internal or private.
+- **Include incidents**: Specify which types of incidents to announce—options include all incidents, drills, or non-drills.
+
+1. Apply filters to your incident announcements to tailor the notifications to specific channels:
+
+- **Incident filter**: For example, you might filter by label, such as `label: 'squad:datasources'`.
+
+### Manage Slack attachments
+
+When you use the 🤖 emoji reaction on a Slack message containing a file, the file is securely copied to Grafana Cloud storage.
+This ensures your incident timeline remains intact, even if the file is later deleted from Slack. Here’s how it works:
+
+- **File retention**: Control attachment retention in your incident timeline. Removing the 🤖 reaction will delete the attached files from Grafana Cloud storage.
+- **Incident web app**: Deleting an item from the timeline within the Incident web app also removes the associated file from Grafana Cloud storage.
+- **File size limit**: Files in Grafana Incident are limited to 100MB. Files too large to persist will still be accessible via a link to the source file in Slack.
+
+## Available Slack commands
+
+{{< admonition type="note" >}}
+The `/escalate` and `/incident` Slack commands have been deprecated. Use `/grafana-irm` to learn more.
+{{< / admonition >}}
+
+The `/grafana-irm` Slack commands allow users and teams to respond to alert groups and collaborate on incidents directly from Slack.
+
+{{< admonition type="tip" >}}
+Use the 🤖 robot emoji Slack reaction to add important messages to the incident timeline.
+And use the 📝 memo emoji Slack reaction to add messages to the resolution notes.
```

Comment:
📝 emoji works differently - it's added to message when user added it to resolution notes via message shortcut or modal. We are planning to rework this behavior to work like 🤖 emoji, but with no estimates.